wizeline/remix-project-lab
# Feat/add contributors path

# What does this PR do?
Adds functionality to adit project's contributors path.


# Where should the reviewer start?
Check the new files and changes added to:
/EditContributorsPath/
/edit.tsx
/editContributorsPath.tsx



# How should this be manually tested?

1. Run the app locally.
2. Go to edit project, and then to Contributors Path section.
3. Try adding, editing and deleting project's stages and tasks
4. Confirm changes are saved properly



# Any background context you want to provide?
We are including the Contributor's Path section to project's information, as users have requested it as feature missing from WizeLab's previous version


# What are the relevant tickets?
#145 


# Screenshots
<img width="1181" alt="Captura de Pantalla 2023-05-11 a la(s) 2 33 49" src="https://github.com/wizeline/remix-project-lab/assets/90728303/58c79c69-019e-427e-9b5a-4729dc28acac">

<img width="1324" alt="Captura de Pantalla 2023-05-11 a la(s) 2 34 05" src="https://github.com/wizeline/remix-project-lab/assets/90728303/b471e333-d603-409d-ac87-c5aa026e9b8c">

<img width="1229" alt="Captura de Pantalla 2023-05-11 a la(s) 2 34 23" src="https://github.com/wizeline/remix-project-lab/assets/90728303/5fa2078f-e8c2-40da-990f-5395a2c3a889">


# Questions



# Checklist
